Error 2021-01-25 03:20:18: DataScience Error [Error: Could NOT open editor for "file:///home/sting/ASSG/LP11/MLAD%20LP%2011%20Project%20movie%20similarity%20blank%20.ipynb".
	at t.ExtHostEditors.showTextDocument (/home/sting/.vscode-server/bin/ea3859d4ba2f3e577a159bc91e3074c5d85c0523/out/vs/server/remoteExtensionHostProcess.js:878:476)
	at processTicksAndRejections (internal/process/task_queues.js:94:5)
	at async T.openNotebookAndCloseEditor (/home/sting/.vscode-server/extensions/ms-toolsai.jupyter-2020.12.414227025/out/client/extension.js:49:399978)]
Error 2021-01-25 03:20:50: Error Loading WebviewPanel: Error: Webview is disposed
Error 2021-01-25 03:20:50: DataScience Error [Error: Webview is disposed
	at S.assertNotDisposed (/home/sting/.vscode-server/bin/ea3859d4ba2f3e577a159bc91e3074c5d85c0523/out/vs/server/remoteExtensionHostProcess.js:896:749)
	at S.get viewColumn [as viewColumn] (/home/sting/.vscode-server/bin/ea3859d4ba2f3e577a159bc91e3074c5d85c0523/out/vs/server/remoteExtensionHostProcess.js:896:179)
	at o.show (/home/sting/.vscode-server/extensions/ms-toolsai.jupyter-2020.12.414227025/out/client/extension.js:47:299569)
	at processTicksAndRejections (internal/process/task_queues.js:94:5)
	at async f.show (/home/sting/.vscode-server/extensions/ms-toolsai.jupyter-2020.12.414227025/out/client/extension.js:9:149682)
	at async T.showEditor (/home/sting/.vscode-server/extensions/ms-toolsai.jupyter-2020.12.414227025/out/client/extension.js:49:404285)
	at async T.open (/home/sting/.vscode-server/extensions/ms-toolsai.jupyter-2020.12.414227025/out/client/extension.js:49:401328)
	at async T.openNotebookAndCloseEditor (/home/sting/.vscode-server/extensions/ms-toolsai.jupyter-2020.12.414227025/out/client/extension.js:49:399955)]
Error 2021-01-25 03:20:50: DataScience Error [Error: Webview is disposed
	at S.assertNotDisposed (/home/sting/.vscode-server/bin/ea3859d4ba2f3e577a159bc91e3074c5d85c0523/out/vs/server/remoteExtensionHostProcess.js:896:749)
	at S.get viewColumn [as viewColumn] (/home/sting/.vscode-server/bin/ea3859d4ba2f3e577a159bc91e3074c5d85c0523/out/vs/server/remoteExtensionHostProcess.js:896:179)
	at o.show (/home/sting/.vscode-server/extensions/ms-toolsai.jupyter-2020.12.414227025/out/client/extension.js:47:299569)
	at processTicksAndRejections (internal/process/task_queues.js:94:5)
	at async f.show (/home/sting/.vscode-server/extensions/ms-toolsai.jupyter-2020.12.414227025/out/client/extension.js:9:149682)
	at async T.showEditor (/home/sting/.vscode-server/extensions/ms-toolsai.jupyter-2020.12.414227025/out/client/extension.js:49:404285)
	at async T.open (/home/sting/.vscode-server/extensions/ms-toolsai.jupyter-2020.12.414227025/out/client/extension.js:49:401328)
	at async T.openNotebookAndCloseEditor (/home/sting/.vscode-server/extensions/ms-toolsai.jupyter-2020.12.414227025/out/client/extension.js:49:399955)]
Error 2021-01-25 03:20:50: DataScience Error [Error: Webview is disposed
	at S.assertNotDisposed (/home/sting/.vscode-server/bin/ea3859d4ba2f3e577a159bc91e3074c5d85c0523/out/vs/server/remoteExtensionHostProcess.js:896:749)
	at S.get viewColumn [as viewColumn] (/home/sting/.vscode-server/bin/ea3859d4ba2f3e577a159bc91e3074c5d85c0523/out/vs/server/remoteExtensionHostProcess.js:896:179)
	at o.show (/home/sting/.vscode-server/extensions/ms-toolsai.jupyter-2020.12.414227025/out/client/extension.js:47:299569)
	at processTicksAndRejections (internal/process/task_queues.js:94:5)
	at async f.show (/home/sting/.vscode-server/extensions/ms-toolsai.jupyter-2020.12.414227025/out/client/extension.js:9:149682)
	at async T.loadNotebookEditor (/home/sting/.vscode-server/extensions/ms-toolsai.jupyter-2020.12.414227025/out/client/extension.js:49:403893)
	at async T.create (/home/sting/.vscode-server/extensions/ms-toolsai.jupyter-2020.12.414227025/out/client/extension.js:49:404430)
	at async T.open (/home/sting/.vscode-server/extensions/ms-toolsai.jupyter-2020.12.414227025/out/client/extension.js:49:401355)
	at async T.openNotebookAndCloseEditor (/home/sting/.vscode-server/extensions/ms-toolsai.jupyter-2020.12.414227025/out/client/extension.js:49:399955)]
Error 2021-01-25 03:22:57: Jupyter Kernel Spec not found for a local connection
> /usr/bin/python3 ~/.vscode-server/extensions/ms-toolsai.jupyter-2020.12.414227025/pythonFiles/pyvsc-run-isolated.py vscode_datascience_helpers.daemon --daemon-module=vscode_datascience_helpers.kernel_launcher_daemon -v
> /usr/bin/python3 ~/.vscode-server/extensions/ms-toolsai.jupyter-2020.12.414227025/pythonFiles/pyvsc-run-isolated.py vscode_datascience_helpers.daemon --daemon-module=vscode_datascience_helpers.kernel_launcher_daemon -v
Started kernel Python 3.8.5 64-bit
> /usr/bin/python3 ~/.vscode-server/extensions/ms-toolsai.jupyter-2020.12.414227025/pythonFiles/pyvsc-run-isolated.py vscode_datascience_helpers.daemon --daemon-module=vscode_datascience_helpers.kernel_launcher_daemon -v
> /usr/bin/python3 ~/.vscode-server/extensions/ms-toolsai.jupyter-2020.12.414227025/pythonFiles/pyvsc-run-isolated.py vscode_datascience_helpers.daemon --daemon-module=vscode_datascience_helpers.kernel_launcher_daemon -v
> ~/.vsch/bin/python ~/.vscode-server/extensions/ms-toolsai.jupyter-2020.12.414227025/pythonFiles/pyvsc-run-isolated.py -c "import ipykernel"
> ~/.vsch/bin/python ~/.vscode-server/extensions/ms-toolsai.jupyter-2020.12.414227025/pythonFiles/pyvsc-run-isolated.py -c "import ipykernel"
Started kernel Python 3.8.5 64-bit ('.vsch')
> ~/.vsch/bin/python ~/.vscode-server/extensions/ms-toolsai.jupyter-2020.12.414227025/pythonFiles/pyvsc-run-isolated.py vscode_datascience_helpers.daemon --daemon-module=vscode_datascience_helpers.kernel_launcher_daemon -v
> ~/.vsch/bin/python ~/.vscode-server/extensions/ms-toolsai.jupyter-2020.12.414227025/pythonFiles/pyvsc-run-isolated.py vscode_datascience_helpers.daemon --daemon-module=vscode_datascience_helpers.kernel_launcher_daemon -v
Error 2021-01-25 03:49:36: Kernel died [r [Error]: [IPKernelApp] ERROR | Exception in message handler:

	at t._next (/home/sting/.vscode-server/extensions/ms-toolsai.jupyter-2020.12.414227025/out/client/extension.js:47:758167)
	at t.__tryOrUnsub (/home/sting/.vscode-server/extensions/ms-toolsai.jupyter-2020.12.414227025/out/client/extension.js:32:5463)
	at t.next (/home/sting/.vscode-server/extensions/ms-toolsai.jupyter-2020.12.414227025/out/client/extension.js:32:4632)
	at t._next (/home/sting/.vscode-server/extensions/ms-toolsai.jupyter-2020.12.414227025/out/client/extension.js:32:3736)
	at t.next (/home/sting/.vscode-server/extensions/ms-toolsai.jupyter-2020.12.414227025/out/client/extension.js:32:3407)
	at t.next (/home/sting/.vscode-server/extensions/ms-toolsai.jupyter-2020.12.414227025/out/client/extension.js:32:8976)
	at /home/sting/.vscode-server/extensions/ms-toolsai.jupyter-2020.12.414227025/out/client/extension.js:9:107791
	at /home/sting/.vscode-server/extensions/ms-toolsai.jupyter-2020.12.414227025/out/client/extension.js:47:421842
	at /home/sting/.vscode-server/extensions/ms-toolsai.jupyter-2020.12.414227025/out/client/extension.js:47:422060
	at Immediate._onImmediate (/home/sting/.vscode-server/extensions/ms-toolsai.jupyter-2020.12.414227025/out/client/extension.js:47:423625)
	at processImmediate (internal/timers.js:439:21)] 
Error 2021-01-25 04:45:21: Disposing session as kernel process died ExitCode: undefined, Reason: undefined
